### DISCUSSION TITLE
Question for Docker-compose.xml

### DISCUSSION THREAD

**Discussion by Ramses7420** [teslamate-org/teslamate#3837] [Q&A]
I just noticed that when I make my backup, I have this message to wander on my Mac terminal. What is this little problem?
<img width="574" alt="Capture d’écran 2024-04-12 à 14 02 33" src="https://github.com/teslamate-org/teslamate/assets/54696668/44b06aba-6fab-4651-8787-1e96e3984cd4">

**Comment by cwanja** [comment]
What version of Docker compose are you running? You might need to upgrade Docker Compose or it could be a bug from this quick search: https://github.com/docker/compose/issues/11628

**Reply by Ramses7420** [reply]
I use the 4.29 version of Docker. I deleted the line "version:" 3 "in docker-compose.yml and everything works

**Comment by StarkITA** [comment]
I have been seeing that as well in new installs of doker-compose, I imagine there is some change on their end, but I did not investigate further.

After all, "WARN" is not "ERR" ;)
